src/substrate-bridge.md

```diff
@@ -0,0 +1,187 @@
+# Substrate bridge
+
+## Main components
+
+SORA 2 interaction with Polkadot/Kusama and other relay chains is visualised in the following diagram:
```

Comment:
```suggestion
SORA interaction with Polkadot/Kusama and other relay chains is visualised in the following diagram:
```

Reply:
I removed the 2 here to avoid raising questions about the numbers and why they're used.